feedthefox/urls.py

```diff
@@ -12,7 +12,8 @@
     '',
     url(r'^$', 'feedthefox.base.views.home', name='home'),
     url(r'^devices/$', 'feedthefox.devices.views.devices', name='devices'),
-    url(r'^device/$', 'feedthefox.devices.views.device', name='device'),
+    url(r'^device/(?P<manufacturer>[\w|\W]+)/(?P<model>[\w|\W]+)$', 'feedthefox.devices.views.device',
```

Comment:
Because we don't have any limitation on the model/manufacturer name and this can be any alphanumeric or any non-alphanumeric.